DH_Engine/Classes/DHGameReplicationInfo.uc

```diff
@@ -792,6 +793,35 @@ simulated function string GetTeamScaleString(int Team)
    }
 }
 
+// Modified to allow VoiceID to be greater than 32
+simulated function AddPRI(PlayerReplicationInfo PRI)
+{
+    local byte      NewVoiceID;
+    local int       i;
+
+    if (Level.NetMode == NM_ListenServer || Level.NetMode == NM_DedicatedServer)
+    {
+        for (i = 0; i < PRIArray.Length; i++)
+        {
+            if (PRIArray[i].VoiceID == NewVoiceID)
+            {
+                i = -1;
+                NewVoiceID++;
+                continue;
+            }
+        }
+
+        if (NewVoiceID >= VOICEID_MAX)
```

Comment:
Why do we have a max voice ID? Why not just let it run up to `255` and then overflow back to `0` if it ever gets that high?